### PR TITLE
publish the library to our internal repo

### PR DESCRIPTION
**Pull request by jflinchbaugh**
Followed examples from our forks of jarchivelib, pecoff4j, etc. In following those examples, this still publishes to RSO, even though we expected they deployed directly to RSC.

It went a little crazy in a little loop bumping its own version, but I'll fix that in the release build: It's a folder now for testing, but shouldn't be.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,18 +19,19 @@
...
+        <groupId>com.sonatype.data.commons</groupId>
+        <artifactId>parent</artifactId>
+        <version>42</version>
+        <relativePath />
     </parent>
 
     <groupId>com.smoketurner</groupId>
     <artifactId>dropwizard-swagger</artifactId>
-    <version>2.0.12-2-sonatype-fork-6</version>
+    <version>2.0.12-2-sonatype-fork-9-SNAPSHOT</version>
```
Comment (jflinchbaugh) [inline pom.xml:30]: it does snapshots automatically now. that's nice.

File: pom.xml [hunk 2]
```diff
@@ -47,12 +48,16 @@
 
     <inceptionYear>2014</inceptionYear>
     <scm>
-        <connection>scm:git:git://github.com:smoketurner/dropwizard-swagger.git</connection>
-        <developerConnection>scm:git:git@github.com:smoketurner/dropwizard-swagger.git</developerConnection>
-        <url>http://github.com/smoketurner/dropwizard-swagger</url>
+        <connection>scm:git:https://github.com/sonatype/dropwizard-swagger.git</connection>
+        <developerConnection>scm:git:https://github.com/sonatype/dropwizard-swagger.git</developerConnection>
+        <url>https://github.com/sonatype/dropwizard-swagger</url>
```
Comment (jflinchbaugh) [inline pom.xml:53]: updated urls so that automatic versions/snapshots worked.

File: pom.xml [hunk 3]
```diff
@@ -47,12 +48,16 @@
...
         <tag>HEAD</tag>
     </scm>
 
+    <properties>
+        <dropwizard.version>2.0.15</dropwizard.version>
+    </properties>
```
Comment (jflinchbaugh) [inline pom.xml:59]: this version and a couple extra dependencies are now here explicitly. it previously came from the dropwizard parent pom.

File: pom.xml [hunk 4]
```diff
@@ -118,6 +128,30 @@
             <artifactId>rest-assured</artifactId>
             <version>4.3.2</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
```
Comment (jflinchbaugh) [inline pom.xml:135]: exclude to remediate the duplicate-class warnings.

File: pom.xml [hunk 5]
```diff
@@ -118,6 +128,30 @@
...
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-testing</artifactId>
+            <version>${dropwizard.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hamcrest</groupId>
+                    <artifactId>hamcrest-core</artifactId>
+                </exclusion>
+            </exclusions>
```
Comment (jflinchbaugh) [inline pom.xml:148]: exclude to remediate the duplicate-class warnings.

**Review by voronaam — APPROVED** [review]

**Review by andrewstein — APPROVED** [review]

**Pull request merged**